plugin/hosts/README.md

```diff
@@ -39,8 +39,11 @@ PTR records for reverse lookups are generated automatically by CoreDNS (based on
 ## Syntax
 
 ~~~
-hosts [FILE [ZONES...]] {
+hosts [no-reverse] [FILE [ZONES...]] {
```

Comment:
Sorry, just now taking a closer look.

Can we not have `no-reverse` here, only in the block below?

Reply:
sure, it removes code. will do.